src/components/Details/VmTemplateDetails/VmTemplateDetails.js

```diff
@@ -145,7 +145,10 @@ export class VmTemplateDetails extends React.Component {
     const { vmTemplate, dataVolumes, NamespaceResourceLink, LoadingComponent } = this.props;
     const id = getId(vmTemplate);
     const vm = selectVm(vmTemplate.objects);
-    const baseTemplateLabels = getVmTemplate(vmTemplate);
+    const getTemplateDisplayName = () => {
```

Comment:
I meant, can we replace the getVmTemplate method? So it can be also reused in vmDetails.

Reply:
:-)

Reply:
I think it's ok now :white_flower: